package.json

```diff
@@ -52,7 +52,7 @@
     "tiptap-vuetify": "^2.24.0",
     "uuid": "^9.0.0",
     "v-sanitize": "^0.0.11",
-    "vue": "^2.7.14",
+    "vue": "2.7.10",
```

Comment:
Something happened in 2.7.11 that caused the build to take much longer and sometimes hang. Supposedly it was fixed... but it still happens in 2.7.14.

Note the lack of `^` so that npm doesn't automatically take the latest "2.x.x" version.

Reply:
should we check other UIs?

Reply:
Yep.